go.sum

```diff
@@ -9,8 +9,9 @@ github.com/apparentlymart/go-textseg/v15 v15.0.0 h1:uYvfpb3DyLSCGWnctWKGj857c6ew
 github.com/apparentlymart/go-textseg/v15 v15.0.0/go.mod h1:K8XmNZdhEBkdlyDdvbmmsvpAG721bKi0joRfFdHIWJ4=
 github.com/bufbuild/protocompile v0.4.0 h1:LbFKd2XowZvQ/kajzguUp2DC9UEIQhIq77fZZlaQsNA=
 github.com/bwesterb/go-ristretto v1.2.3/go.mod h1:fUIoIZaG73pV5biE2Blr2xEzDoMj7NFEuV9ekS419A0=
-github.com/cloudflare/circl v1.3.3 h1:fE/Qz0QdIGqeWfnwq0RE0R7MI51s0M2E4Ga9kq5AEMs=
 github.com/cloudflare/circl v1.3.3/go.mod h1:5XYMA4rFBvNIrhs50XuiBJ15vF2pZn4nnUKZrLbUZFA=
```

Comment:
This should be deleted now?

Reply:
Eh, not exactly. This change was not made manually, it's made by running `go get github.com/cloudflare/circl` (to upgrade the dependency) and `go mod tidy` (to "clean" the dependency files). `go.sum` is a checksum file (not a lock file) and it can include checksums of versions that will not eventually be used for the build. It probably means that some transitive dependency asks for circl v1.3.3 (and so the version might be inspected), but it's ultimately rejected. In any case, removing that line and running `go mod tidy` will insert it again.